tests/factories/PacbioTagSetFactory.js

```diff
@@ -23,6 +23,7 @@ const createStoreData = (data) => {
         first: (n = 1) => tagSet.tags.slice(0, n).map((id) => tags[id]),
       },
     },
+    findTagSetByName: (name) => Object.values(tagSets).find((ts) => ts.name === name),
```

Comment:
These are great but wondering if we are doing this in the actual code and if so should it be a function elsewhere?

Reply:
I don't think we currently have a similar function elsewhere.